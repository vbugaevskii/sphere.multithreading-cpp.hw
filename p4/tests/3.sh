true&&echo 1-true 
false&&echo 1-false
true || echo 2-true
false   ||echo 2-false

true&&false&&echo 3-1
true||false&&echo 3-2

echo 123 && echo 456   &&echo 789

echo 123 > file.file && cat < file.file > file1.file && cat file1.file file1.file
