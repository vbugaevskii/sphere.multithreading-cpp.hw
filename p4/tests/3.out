1-true
2-false
3-2
123
456
789
123
123
